src/coreclr/inc/check.h

```diff
@@ -532,7 +532,7 @@ CHECK CheckValue(TYPENAME &val)
 
 #endif // DACCESS_COMPILE
 
-#endif // _PREFAST_ || _PREFIX_
+#endif
 
 #define COMPILER_ASSUME(_condition) \
     COMPILER_ASSUME_MSG(_condition, "")
```

Comment:
Delete PREFIX_ASSUME and PREFAST_ASSUME below as well?

Reply:
Done.